tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py

```diff
@@ -171,8 +177,13 @@ def render_routine(line):
         if not os.path.exists(pdf_filename):
             logging.info('ERROR: %s cannot compile\n', file_idx)
         else:
-            subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
-                            pdf_filename, png_filename], check=True, stdout=PIPE, stderr=PIPE)
+            if os.name == 'nt':
+                convert_command = subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
```

Comment:
please check command status and handle errors or change on check=True (it made this automatically)

Reply:
Added `shell=True`. It was changed on the debugging purpose, but later I forgot to change it back.